qutip/visualization.py

```diff
@@ -852,8 +894,8 @@ def matrix_histogram_complex(M, xlabels=None, ylabels=None,
     return fig, ax
 
 
-def plot_energy_levels(H_list, N=0, labels=None, show_ylabels=False,
-                       figsize=(8, 12), fig=None, ax=None):
+def plot_energy_levels(H_list, h_labels=None, energy_levels=None, N=0, *,
```

Comment:
Improvement that could come later: scale should be larger than the first Hamiltonian.

Reply:
Sorry. What do you mean by that? I hope you add details with this plot.
<img width="594" alt="plot_energy_levels" src="https://github.com/qutip/qutip/assets/72233550/3bb7eee7-d346-4be6-a20b-82e2c5e83b67">

Reply:
This scale `[-2, 0, 2]` on the left is not great for explaining the second set of values which range is wider.
it get harder to read the more Hamiltonian there are.
Adding another scale on the right could be useful.
I do not expect you to fix this there.

Reply:
I tried, but they do not look good. I will not do it now.